mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/domain/usecase/GenerateQr.kt

```diff
@@ -0,0 +1,61 @@
+package org.mifos.mobilewallet.mifospay.qr.domain.usecase
+
+import android.graphics.Bitmap
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.MultiFormatWriter
+import com.google.zxing.WriterException
+import com.google.zxing.common.BitMatrix
+import org.mifos.mobilewallet.core.base.UseCase
+import org.mifos.mobilewallet.mifospay.utils.Constants
+import javax.inject.Inject
+
+/**
+ * Created by naman on 8/7/17.
+ */
+open class GenerateQr @Inject constructor() :
+    UseCase<GenerateQr.RequestValues?, GenerateQr.ResponseValue?>() {
+    override fun executeUseCase(requestValues: RequestValues?) {
```

Comment:
Same here, for executing the usecase request value is definitely needed so no need for `?` here